src/main/java/dk/cngroup/lentils/controller/ImageController.java

```diff
@@ -0,0 +1,37 @@
+package dk.cngroup.lentils.controller;
+
+import dk.cngroup.lentils.service.ImageService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.MultipartFile;
+
+@Controller
+@RequestMapping("/admin/image")
+public class ImageController {
+    private static final String VIEW_IMAGE_FORM = "image/form";
+    private static final String VIEW_IMAGE_DETAIL = "image/detail";
+
+    private ImageService imageService;
+
+    @Autowired
+    public ImageController(final ImageService imageService) {
+        this.imageService = imageService;
+    }
+
+    @GetMapping(value = "/")
+    public String displayForm() {
+        return VIEW_IMAGE_FORM;
+    }
+
+    @PostMapping(value = "/uploadImage")
+    public String submit(@RequestParam("file") final MultipartFile file, final Model model) {
+        model.addAttribute("file", file);
+        imageService.readAndSaveImageFile(file);
```

Comment:
Asi staci `imageService.save();` To precteni pak bude v metode, ktera bude prijimat ten MulripartFile parametr.